bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestRpcRetryBehavior1x.java

```diff
@@ -0,0 +1,167 @@
+package com.google.cloud.bigtable.hbase;
```

Comment:
This test is the main work in progress:

* Still looking to test different methods
* Will see how to add this for the HBase 2.x client too